README.md

```diff
@@ -92,6 +128,61 @@ Install the requirements mentioned above and copy the distribution into
 any directory you like.
 
 
+Execute
+-------
+
+```
+python2 ../ci-dnf-stack/cidnfstack.py build -c dnf-pull-requests tito
+```
+Localtest (omly for project that use tito):
+```
+python2 /path/to/cidnfstack.py build
+```
+Will test the project in current directory.
+
+Or:
+```
+python2 /path/to/cidnfstack.py build -l /path/to/project/1/ -l /path/to/project/2/
+```
+Will test projects from directories described by -l option. It will inside docker container install all requirements
+for build, than it builds project 1 and install it in container, than it is repeated for second project.
```

Comment:
To the environment (docker container) will be installed dnf eco-system nightly projects from COPR rpmsoftwaremanagement/dnf-nightly, then it will install all requirements for 1. project and install the project itself. The same proces will be repetated for project 2 and so on.